orchestration/dagster/hca_orchestration/solids.py

```diff
@@ -0,0 +1,61 @@
+from dagster import solid, Nothing, InputDefinition, String
+
+STAGING_BUCKET_NAME = "staging_bucket_name"
+STAGING_PREFIX_NAME = "staging_prefix_name"
+
+
```

Comment:
as a somewhat generic comment, I foresee an explosion in the number of solids and a possible desire to split them up into multiple files at some point down the line

Reply:
there's a few architectural approaches we can take there, but that shouldn't be difficult to do as needed later.